net/browserless_agent/src/browserless_agent.ts

```diff
@@ -0,0 +1,254 @@
+import { AgentFunction, AgentFunctionInfo, DefaultConfigData } from "graphai";
+
+interface BrowserlessInputs {
+  url: string;
+  operation?: "content" | "screenshot";
+  options?: Record<string, any>;
+  headers?: Record<string, string>;
+  cookies?: Array<{
+    name: string;
+    value: string;
+    domain?: string;
+  }>;
+}
+
+// Type for the buffer response from screenshot operations
+interface BufferResponse {
+  base64: string;
+  contentType: string;
+}
+
+interface BrowserlessParams {
+  debug?: boolean;
+  type?: "json" | "text" | "buffer";
+  throwError?: boolean;
+}
+
+type BrowserlessResult =
+  | object
+  | string
+  | BufferResponse
+  | {
+      onError?: {
+        message: string;
+        status?: number;
+        error: any;
+      };
+    };
+
+export const browserlessAgent: AgentFunction<BrowserlessParams, BrowserlessResult, BrowserlessInputs, DefaultConfigData> = async ({ namedInputs, params }) => {
+  const { url, operation, options, headers, cookies } = namedInputs;
+  const throwError = params?.throwError ?? false;
+
+  const browserlessEndpoint = "https://chrome.browserless.io";
+  const browserlessToken = process.env.BROWSERLESS_API_TOKEN;
+
+  // Check if API token is provided
+  if (!browserlessToken) {
+    const errorMessage = "Browserless API token is required. Please set the BROWSERLESS_API_TOKEN environment variable.";
+    throw new Error(errorMessage);
+  }
+
+  // Build the endpoint with token
+  const endpoint = `${browserlessEndpoint}?token=${browserlessToken}`;
+
+  // Select appropriate endpoint based on operation type
+  const operationType = operation || "content";
```

Comment:
You are using both ?? and ||, but if there is no specific reason for differentiating them, please standardize on ??.